src/main/webapp/resources/js/pages/projects/linelist/components/Table/renderers/IconCellRenderer.jsx

```diff
@@ -13,7 +13,9 @@ function LockedIcon() {
 }
 
 export class IconCellRenderer extends React.Component {
-  static propTypes = {};
+  static propTypes = {
+    data: PropTypes.func.isRequired
```

Comment:
Should be PropTypes.object.isRequired instead of PropTypes.func.isRequired

Reply:
Fixed in eaf28cf4526f28ae51856d3a8d1c66896137fa25